CMakeLists.txt

```diff
@@ -111,7 +111,11 @@ if(BUILD_TESTING)
   if(TARGET benchmark_log)
     target_include_directories(benchmark_log PRIVATE ${console_bridge_INCLUDE_DIRS})
     target_link_libraries(benchmark_log ${console_bridge_LIBRARIES})
-    target_link_directories(benchmark_log PRIVATE ${console_bridge_LIBRARY_DIRS})
+    if(${CMAKE_VERSION} VERSION_LESS "3.13")
+      link_directories(${console_bridge_LIBRARY_DIRS})
+    else()
+      target_link_directories(benchmark_log PRIVATE ${console_bridge_LIBRARY_DIRS})
+    endif()
```

Comment:
I'm just curious; is this needed at all?  When I ran it locally commenting out `target_link_directories`, it seemed to build and run just fine for me.

Reply:
This is need it when you are trying to link against `console_bridge` but the install hasn't happend yet. For example: when the workspace is clean or you are compiling for the first time this package

Reply:
All right, thanks for the explanation.